main.py

```diff
@@ -21,7 +21,7 @@ def parallel_feature_extraction(args):
     elif args.feature_type == 'vggish':
         from models.vggish.extract_vggish import ExtractVGGish  # defined here to avoid import errors
         extractor = ExtractVGGish(args)
-    elif args.feature_type in ['resnet18', 'resnet34', 'resnet50', 'resnet101', 'resnet152']:
```

Comment:
Thanks for this and the rest of the resnet changes! I am happy to have them.

One comment though, please try to avoid doing multiple things in a single PR. It is considered as a bad practice since the branch is about clip and having changes of the resnet features could be misleading.

I would create a separate PR changing resnet logic instead. 

But that's ok, we can merge these two things, it is not a big deal.

Reply:
I'm sorry :fearful:
I will avoid that next time

Reply:
No worries!